controller/controller_client.go

```diff
@@ -2,7 +2,7 @@
 // or more contributor license agreements. Licensed under the Elastic License;
 // you may not use this file except in compliance with the Elastic License.
 
-package server
+package controller
```

Comment:
👍 Moving this out of the server package is a good idea. It makes the boundaries between the agent interaction and the shipper implementation much clearer.